public/css/list/redundancy-group-list-item.less

```diff
@@ -0,0 +1,20 @@
+.redundancy-group-list-item {
+  .caption {
+    display: flex;
+    justify-content: space-between;
```

Comment:
Instead of this, and because you already have a rule for plugin output, why not making the latter `flex: 1 1 auto`?

Reply:
The parent element `main` has `display: block` property, `flex: 1 1 auto` does not work. And changing the `display` property in  `main` to `flex` could have other consequences.